### ISSUE TITLE
[Bug]: Accessibility Issues with Items that Expand, and Missing Labels

### ISSUE THREAD

**Issue by duplaja** [advplyr/audiobookshelf#441]
### Describe the issue

Items that can expand when clicked, such as: 

* filter dropdown
* sort dropdown
* chapter selection from the player

are hard-coded with aria-expanded="true". This should be false when they are not opened up, and true when they are expanded.

In addition, these items are all missing properly accessible labels.

They are using the aria-labeledby attribute which is supposed to reference another element that has the correct label. However, these all are set as "listbox-label", which is not actually an element that exists on the page. If you want to label these directly, these should use aria-label="<Name of Item>".

### Steps to reproduce the issue

1. Appears when using Audiobookshelf site with a screen reader of any sort.

### Audiobookshelf version

v1.7.2

### How are you running audiobookshelf?

Debian/PPA

**Comment by duplaja** [comment]
![library-screen](https://user-images.githubusercontent.com/20245408/161409833-8d8efe6c-4376-429f-bb53-65bb6bce9684.png)

This is the aria view of the library screen, for example. I've submitted a pull request that should address the add / remove at the bottom (those are the adjust the ebook cover size), and partially address the account label (need to probably remove "root", and the pull request replaces "person" with "User Account").

You can see that the Filter just shows as "All", with no other label, and as expanded even when it first loads. Similarly, "Sort" appears as "Title expand_less".

One good resource for (quickly) checking what a screenreader will see, as far as labels, is [this Chrome extension](https://chrome.google.com/webstore/detail/aria-devtools/dneemiigcbbgbdjlcdjjnianlikimpck/related?hl=en)
